physics/module_MYNNPBL_wrapper.meta

```diff
@@ -27,7 +27,6 @@
   intent = out
   optional = F
 
-#####################################################################
```

Comment:
This line helps to spot that this is a new metadata table within the same file

Reply:
It seems reasonable to add that line. I don't recall removing it. I must be schizophrenic.